kernel/src/misc/wm/wm.c

```diff
@@ -469,8 +473,8 @@ void wm_draw_mouse(rect_t new) {
  */
 void wm_mouse_callback(mouse_t raw_curr) {
     static mouse_t raw_prev;
-    static wm_window_t* dragged = NULL;
-    static bool been_dragged = false;
+    static wm_window_t* dragged = NULL,* hovered = NULL;
+    static uint16_t been_dragged = false;
```

Comment:
Thy may was a faulty act. I'm not used to bool type in C (even in my own sources I end up using u16 instead of bool sometimes). No reason at all for it. I may have happened because I removed then re-added the variable